.github/workflows/build-and-test.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
       - dev
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
I assume `synchronize` includes new pushes to the PR? Just wondering if this needs `edited` too (I assume that actually means the description is edited not the content but I couldn't find the right documentation).

What scenario do these conditions prevent by the way?

Reply:
`open`, `synchronize` and `reopened` are equivalent to just having `pull_request` (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request). So the change to before is the addition of `ready_for_review`. It sucks that can't go "default + `ready_for_review`", but I don't think the defaults would change much.